boards/st/stm32h745i_disco/doc/index.rst

```diff
@@ -67,7 +67,7 @@ More information about STM32H747XIH6 can be found here:
 Supported Features
 ==================
 
-The current Zephyr stm32h745i_disco board configuration supports the following hardware features:
+The current Zephyr ``stm32h745i_disco`` board target supports the following hardware features:
```

Comment:
There are 2 targets for this board.

Reply:
Thanks, I've added this to the opt-out list.